lib/posix/stropts.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024 Abhinav Srivastava
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/posix/stropts.h>
+#include <errno.h>
+#include <zephyr/kernel.h>
+
```

Comment:
```suggestion

/**
 * @brief Send a message on a STREAM.
 *
 * See IEEE 1003.1
 */
```
i.e.: 
https://github.com/zephyrproject-rtos/zephyr/blob/114548ef607aefd712dda7e5ea954981e778185b/lib/posix/pthread.c#L275-L280

Reply:
@AbhinavMir

Reply:
I wouldn't even worry about it all of the doxygen needs to be fixed at some point.